polyfill/lib/time.mjs

```diff
@@ -255,13 +255,14 @@ export class Time {
     let hour, minute, second, millisecond, microsecond, nanosecond;
     if (typeof item === 'object' && item) {
       if (ES.IsTemporalTime(item)) {
+        hour = GetSlot(item, HOUR);
         minute = GetSlot(item, MINUTE);
         second = GetSlot(item, SECOND);
         millisecond = GetSlot(item, MILLISECOND);
         microsecond = GetSlot(item, MICROSECOND);
         nanosecond = GetSlot(item, NANOSECOND);
       } else {
-        // Intentionally alphabetical
+        // Intentionally largest to smallest units
```

Comment:
For future reference, we can just delete this comment, but no need to block the PR on it. (Calling the property getters on a passed-in object in alphabetical order is mandated by the spec, which is now implemented in `ES.ToTemporalTimeRecord`. Here, they can be in any old order.

Reply:
> Calling the property getters on a passed-in object in alphabetical order is mandated by the spec

OK, got it. Out of curiosity, why is alpha order required?

Reply:
Alphabetical doesn't really matter, but some order is required to be specified, since the calls are observable by client code if you pass in a Proxy, for example. I'm not sure why alphabetical was chosen originally as the arbitrary order, but I'd guess it's because there's some precedent elsewhere.